gke-tpu-examples/single-host-inference/jax/stable-diffusion/README.md

```diff
@@ -0,0 +1,6 @@
+## Serve (online inference) a model using a single TPU and GKE 
+
+To better understand how TPUs work on GKE, please read the doc
+[TPUs in GKE introduction](https://cloud.google.com/tpu/docs/tpus-in-gke).
+
+This directory contains files for [JAX Model inference and serving](https://cloud.google.com/tpu/docs/tpus-in-gke#jax-model). You can find step-by-step instructions in the quickstart guide.
```

Comment:
I think the repo should be self sufficient, it is not necessary to add step-by-step instructions, but something workable as I suggested in the other comment. Will the guide be made available on gcp's public documentation page or will it stay in a google doc? If the former, then I guess it is sufficient to link to the quickstart guide once we publish it.

Reply:
Yes, the doc will be published to the public as a quickstart. I plan to link it back once it's available.